chain-config/src/validator_rules.rs

```diff
@@ -104,6 +148,9 @@ pub trait ValidatorRulesModule: setup_phase::SetupPhaseModule + events::EventsMo
     #[storage_mapper("lastBlsKeyId")]
     fn last_bls_key_id(&self) -> SingleValueMapper<BigUint<Self::Api>>;
 
+    #[storage_mapper("genesisPhase")]
+    fn genesis_phase(&self) -> SingleValueMapper<bool>;
```

Comment:
I don't think this naming is correct. It should be like `registration_status` or something similar (enabled/disable)